<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Solution View</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Graphics View</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGraphicsViewWithMouseEvent" name="graphicsView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Text View</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew_Solution"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose_File"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Template</string>
    </property>
    <widget class="QMenu" name="menuEdit_2">
     <property name="title">
      <string>Edit...</string>
     </property>
     <addaction name="actionAdd_Mode"/>
     <addaction name="actionEdit_Mode"/>
     <addaction name="actionRemove_Mode"/>
    </widget>
    <addaction name="actionTemplate"/>
    <addaction name="actionDelete_Template"/>
    <addaction name="menuEdit_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionTemplate">
   <property name="text">
    <string>New Template</string>
   </property>
  </action>
  <action name="actionDelete_Template">
   <property name="text">
    <string>Delete Template</string>
   </property>
  </action>
  <action name="actionAdd_Mode">
   <property name="text">
    <string>Add Mode</string>
   </property>
  </action>
  <action name="actionEdit_Mode">
   <property name="text">
    <string>Edit Mode</string>
   </property>
  </action>
  <action name="actionRemove_Mode">
   <property name="text">
    <string>Remove Mode</string>
   </property>
  </action>
  <action name="actionNew_Solution">
   <property name="text">
    <string>New Solution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGraphicsViewWithMouseEvent</class>
   <extends>QGraphicsView</extends>
   <header location="global">qgraphicsviewwithmouseevent.h</header>
   <slots>
    <signal>MousePressEvent(QMouseEvent*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
